--- a/data/unitytweak.ui
+++ b/data/unitytweak.ui
@@ -361,36 +361,6 @@
                 <property name="homogeneous">True</property>
               </packing>
             </child>
-            <child>
-              <object class="GtkToolItem" id="tool_searchbox">
-                <property name="visible">True</property>
-                <property name="can_focus">False</property>
-                <child>
-                  <object class="GtkEntry" id="tool_entry_search">
-                    <property name="visible">True</property>
-                    <property name="can_focus">True</property>
-                    <property name="has_focus">True</property>
-                    <property name="tooltip_markup" translatable="yes">Search Unity Tweak Tool</property>
-                    <property name="tooltip_text" translatable="yes">Search Unity Tweak Tool</property>
-                    <property name="margin_right">10</property>
-                    <property name="hexpand">True</property>
-                    <property name="max_length">20</property>
-                    <property name="invisible_char">•</property>
-                    <property name="activates_default">True</property>
-                    <property name="width_chars">25</property>
-                    <property name="caps_lock_warning">False</property>
-                    <property name="primary_icon_name">edit-find-symbolic</property>
-                    <signal name="insert-text" handler="on_tool_entry_search_insert_text" swapped="no"/>
-                    <signal name="delete-text" handler="on_tool_entry_search_delete_text" swapped="no"/>
-                    <signal name="icon-press" handler="on_tool_entry_search_icon_press" swapped="no"/>
-                  </object>
-                </child>
-              </object>
-              <packing>
-                <property name="expand">False</property>
-                <property name="homogeneous">True</property>
-              </packing>
-            </child>
           </object>
           <packing>
             <property name="expand">False</property>
